src/Database/PostgreSQL/Simple/Time/Internal/Printer.hs

```diff
@@ -121,3 +125,11 @@ nominalDiffTime :: NominalDiffTime -> Builder
 nominalDiffTime xy = integerDec x <> primBounded frac (abs (fromIntegral y))
   where
     (x,y) = fromPico (unsafeCoerce xy) `quotRem` 1000000000000
+
+calendarDiffTime :: CalendarDiffTime -> Builder
+calendarDiffTime = byteString
+  . fromString
+    -- from the docs: "Beware: fromString truncates multi-byte characters to octets".
+    -- However, I think this is a safe usage, because ISO8601-encoding seems restricted
+    -- to ASCII output.
+  . iso8601Show
```

Comment:
I decided to avoid too much logic to do with `Builder`s and `Parser`s for this type, I thought it would be more maintainable in the long-term to trust the `time` package's out-of-the-box ISO formatter where possible.